channels/ar.m3u

```diff
@@ -42,17 +42,17 @@ https://live-edge01.telecentro.net.ar/live/26hd-720/playlist.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Spanish" tvg-logo="" group-title="News",Canal 26
 https://live-edge01.telecentro.net.ar/live/smil:c26.smil/playlist.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Spanish" tvg-logo="http://tvonlinegratis.com/wp-content/uploads/2014/07/Canal4-SanJuan-FDG.jpg" group-title="",Canal 4 San Juan
-http://streamyes.alsolnet.com:1935/canal4sanjuan/live/chunklist.m3u8
+https://streamyes.alsolnet.com/canal4sanjuan/live/playlist.m3u8
```

Comment:
New link does not work, original link works.

Reply:
This works for me via VLC

Reply:
I'm happy to revert the https removal if that's what the maintainers want.